src/applications/vaos/reducers/expressCare.js

```diff
@@ -126,7 +127,16 @@ export default function expressCareReducer(state = initialState, action) {
           facilityId: facility.id,
           days: facility.customRequestSettings
             .find(setting => setting.id === EXPRESS_CARE)
-            .schedulingDays.filter(day => day.canSchedule),
+            .schedulingDays.filter(day => day.canSchedule)
+            .map(daySchedule => ({
+              ...daySchedule,
+              dayOfWeekIndex: Number(
+                moment()
```

Comment:
What do you think about creating an object that maps the name of the day to the number? This feels like a roundabout way to get it. And having `moment()` in a reducer still seems questionable to me, even though it technically doesn't violate Redux guidelines in this case.

Reply:
Created an object in `constants.js` to get these indexes so we dont' have to use moment.